update-cargo-sources.sh

```diff
@@ -0,0 +1,6 @@
+# must have a checkout from the cigale app of the right version in ../cigale
+mkdir .cargo
```

Comment:
The repository should contain only the files needed to package the app itself. Mind removing this?

Reply:
So a good practice would be to have that script in the main Cigale repo? I'm ok to move it there if it's considered a good practice

Reply:
That's up to you really, you can read more about what should/shouldn't be part of the Flathub repository https://github.com/flathub/flathub/wiki/App-Maintenance#required-files